examples/ping6d/main.rs

```diff
@@ -0,0 +1,58 @@
+use nb2::packets::{
+    icmp::v6::{EchoReply, EchoRequest, Icmpv6},
```

Comment:
did we want this approach going forward? not sure if it's the standard?

Reply:
yeah I saw this somewhere and decided to try it out for just packets cuz that's 90% of the things we typically import. doesn't really feel like it's better than the more standard way though. I will change them back.